src/Entity/IdeasWorkshop/Note.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\EntityIdentityTrait;
+use AppBundle\Entity\EntityNameSlugTrait;
+use AppBundle\Entity\EntityTimestampableTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ *
+ * @ORM\Table(
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="note_slug_unique", columns="slug")
+ *     }
+ * )
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Note
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+    use EntityNameSlugTrait;
+
+    private const PUBLISHED_INTERVAL = 'P3W';
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Theme")
+     */
+    private $theme;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Scale")
+     */
+    private $scale;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="Need")
+     */
+    private $needs;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Adherent")
+     */
+    private $adherent;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $publishedAt;
+
+    /**
+     * @var Committee
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Committee")
+     */
+    private $committee;
+
+    /**
+     * @var ArrayCollection
+     *
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\IdeasWorkshop\Guideline")
+     */
+    private $guidelines;
+
+    /**
+     * @ORM\Column(length=11, options={"default": "IN_PROGRESS"})
+     */
+    private $status;
+
+    public function __construct()
+    {
+        $this->needs = new ArrayCollection();
+        $this->guidelines = new ArrayCollection();
+    }
+
+    public static function create(
+        UuidInterface $uuid,
+        string $name,
+        string $status = NoteStatusEnum::IN_PROGRESS
+    ): Note {
+        $note = new self();
+
+        $note->uuid = $uuid;
+        $note->setName($name);
```

Comment:
Same here, you can set directly the property.

Reply:
:heavy_check_mark: